src/services/azblob/backend.rs

```diff
@@ -577,6 +596,69 @@ impl Accessor for AzblobBackend {
 
         Ok((RpScan::default(), op))
     }
+
+    async fn batch(&self, args: OpBatch) -> Result<RpBatch> {
+        let ops = args.into_operation();
+        match ops {
+            BatchOperations::Delete(ops) => {
+                let paths = ops.into_iter().map(|(p, _)| p).collect::<Vec<_>>();
+                // vector to store all the results
+                let mut results = Vec::with_capacity(paths.len());
+
+                let batches = paths.chunks(AZBLOB_BATCH_LIMIT);
```

Comment:
Please return error directly, we should never loop inside Accessor.

Reply:
Maybe we can offer a `batch_limit` method for `Accessor`s.

Reply:
OK, we do have this method...